src/agent/Cargo.toml

```diff
@@ -71,10 +71,10 @@ openssl = { version = "0.10.38", features = ["vendored"] }
 
 # Image pull/decrypt
 [target.'cfg(target_arch = "s390x")'.dependencies]
-image-rs = { git = "https://github.com/confidential-containers/image-rs", rev = "v0.4.0", default-features = false, features = ["kata-cc-s390x"] }
+image-rs = { git = "https://github.com/confidential-containers/image-rs", rev = "e7f57a4", default-features = false, features = ["kata-cc-s390x"] }
 
 [target.'cfg(not(target_arch = "s390x"))'.dependencies]
-image-rs = { git = "https://github.com/confidential-containers/image-rs", rev = "v0.4.0", default-features = true, features = ["kata-cc", "signature-simple"] }
+image-rs = { git = "https://github.com/confidential-containers/image-rs", rev = "e7f57a4", default-features = false, features = ["kata-cc"] }
```

Comment:
here too

Reply:
sample signing is now included in the big feature `kata-cc` now. Here https://github.com/confidential-containers/image-rs/blob/main/Cargo.toml#L75